.github/workflows/ci.yml

```diff
@@ -11,12 +11,12 @@ jobs:
       - run: echo "The ${{ github.ref }} branch job was automatically triggered by a ${{ github.event_name }} event"
       - name: Clone repo
         uses: actions/checkout@v2
-      - name: Install
-        run: |
-          npm install
-      - name: Build
-        run: |
-          npm run build
+      - name: E2E
```

Comment:
A few questions here:

1. Won't we still need to run an npm install?
2. I think running the build as an explicit step will help catch some errors we might not have noticed. What's the impetus for removing it?
3. I admittedly haven't used  GitHub Action's Cypress runner before. Are there perf benefits here over using the normal `npm run e2e` like the humans will?

Reply:
Looking at the documentation of cypress-io/github-action@v2, it seems to me we actually need to explicitly add a build step (see https://github.com/cypress-io/github-action#build-app) -- at least that's my interpretation.

Something like...
```
- name: Cypress run
        uses: cypress-io/github-action@v2
        with:
          build: npm run build
```

From the documentation it seems that the `install` step happens implicitly as part of the action (probably since it has built-in support for caching).

Reply:
Perhaps you can chip in as well @adomas-aion?

Reply:
I generally prefer explicit commands over things that happen implicitly. If we moved the E2E step, for example, then things would begin failing.